assets/src/edit-story/components/uploadDropTarget/message.js

```diff
@@ -65,11 +65,8 @@ const Icon = styled(UploadIcon)`
 `;
 
 function UploadDropTargetMessage({ message, ...rest }) {
-  const { allowedMimeTypes = {} } = useConfig();
-  const allowedFileTypes = Object.keys(allowedMimeTypes).reduce(
-    (acc, cur) => [...acc, ...allowedMimeTypes[cur]],
-    []
-  );
+  const { allowedFileTypes = {} } = useConfig();
```

Comment:
```suggestion
  const { allowedFileTypes = [] } = useConfig();
```

Only arrays have a `join` method, not objects. So the default value here should be an empty array.

Reply:
Done